draft-ietf-tls-tls13.md

```diff
@@ -2139,13 +2144,13 @@ Clients MUST verify that the server's selected_identity is within the
 range supplied by the client and that the "key_share" and
 "signature_algorithms" extensions are consistent with the
 indicated ke_modes and auth_modes values. If these values
-are not consistent, the client MUST generate an "illegal_parameter"
-alert and close the connection.
+are not consistent, the client MUST abort the handshake with
+a "illegal_parameter" alert.
```

Comment:
Actually, what I believe happened here is that I did s/fatal "/"/ and so there were a pile of 'a fatal "unsupported...' or whatever, and they need to be 'an "unsupported...'